stac_fastapi/pgstac/stac_fastapi/pgstac/version.py

```diff
@@ -0,0 +1,2 @@
+"""library version."""
+__version__ = "0.1.0"
```

Comment:
Should this be synced with other modules' versions? Looks like `1.1.0` elsewhere?

https://github.com/stac-utils/stac-fastapi/blob/c3dd27d0a2f5129f36904dd521d7bff22460c306/stac_fastapi/api/stac_fastapi/api/version.py#L1-L2

Reply:
I think that's a wider question - should each module have it's own version or should we version all of stac-fastapi together? If together, we should try to put the version in one place that we use in all the modules.